src/components/OperationsList/Operation.js

```diff
@@ -49,6 +49,10 @@ const OperationRaw = styled(Box).attrs({
 `
 
 const Address = ({ value }: { value: string }) => {
+    if (!value) {
```

Comment:
there is an issue in term of Flowtype at some point. according to flow `!value` should never have happened. which means props probably should be `{value: ?string}` and then we need also to have that reflected in the original Operation type (not exactly sure where, address autorized to be null ?)

anyway, that's for later, we'll just merge this first ;)

Reply:
https://github.com/LedgerHQ/ledger-live-common/blob/master/src/types/operation.js#L18